lib/tether/ops_linux.go

```diff
@@ -492,7 +506,10 @@ func (t *BaseOperations) updateNameservers(endpoint *NetworkEndpoint) error {
 		log.Infof("Added nameserver: %s", gw.IP)
 	}
 
-	if err := Sys.ResolvConf.Save(); err != nil {
+	if err := Sys.ResolvConf.Save(resolvConfPathBindSrc); err != nil {
```

Comment:
You have these two calls everywhere; why don't you combine them into one helper function?